### PR TITLE
modify get-put-delete.md and cas.md

### PR DESCRIPTION
**Pull request by Liuxiaozhen12**
Signed-off-by: Liuxiaozhen12 <liuxiaozhen@pingcap.com>

### What is changed?

modify get-put-delete.md and cas.md
### Any related PRs or issues?
https://github.com/tikv/website/pull/243

### CODE REVIEW

**Comment by Liuxiaozhen12** [comment]
@marsishandsome  @TomShawn  PTAL, thanks!

**Review by marsishandsome — APPROVED** [review]
LGTM

File: content/docs/5.1/develop/rawkv/cas.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ if (prevValue == request.prevValue) {
 return prevValue;
 ```
 
-The atomicity guarantees that the new value is calculated based on up-to-date information; if the value had been updated by another thread in the meantime, the write would fail.
+The atomicity guarantees that the new value is calculated based on up-to-date information. If the value had been updated by another thread in the meantime, the write would fail.
 
 {{< warning >}}
-CAS can normally prevent problems from concurrent access, but suffers from [ABA problem](https://en.wikipedia.org/wiki/ABA_problem). 
+CAS can normally prevent problems from concurrent access, but suffers from [ABA problem](https://en.wikipedia.org/wiki/ABA_problem).
```
Comment (TomShawn) [inline content/docs/5.1/develop/rawkv/cas.md:25]:
```suggestion
Normally, CAS can prevent problems from the concurrent access, but suffers from the [ABA problem](https://en.wikipedia.org/wiki/ABA_problem).
```

File: content/docs/5.1/develop/rawkv/cas.md [hunk 2]
```diff
@@ -19,15 +19,15 @@ if (prevValue == request.prevValue) {
...
 {{</ warning >}}
 
 ## Java
 
-The following example shows a simple example of how to use the `CAS` API in java.
+The following example shows how to use `CAS` API in java.
```
Comment (TomShawn) [inline content/docs/5.1/develop/rawkv/cas.md:30]: @Liuxiaozhen12 java --> Java    The same for similar places (not in code) in this PR.

File: content/docs/5.1/develop/rawkv/cas.md [hunk 3]
```diff
@@ -70,9 +70,9 @@ session.close();
 ```
 
 {{< warning >}}
-Users must set `conf.setEnableAtomicForCAS(true)` to ensure linearizability of `CAS` when used together with `put`, `delete`, `batch_put`, or `batch_delete`.
+You must set `conf.setEnableAtomicForCAS(true)` to ensure linearizability of `CAS` when using with `put`, `delete`, `batch_put`, or `batch_delete`.
 
-To guarantee the atomicity of CAS, write operations like `put` or `delete` in atomic mode are more expensive.
+To guarantee the atomicity of CAS, operations like `put` or `delete` in atomic mode are more expensive.
```
Comment (TomShawn) [inline content/docs/5.1/develop/rawkv/cas.md:75]:
```suggestion
To guarantee the atomicity of CAS, write operations like `put` or `delete` in atomic mode are more expensive.
```

File: content/docs/5.1/develop/rawkv/get-put-delete.md [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
-This document walks you through how to use RawKV's basic operations such as `Get`, `Put` and `Delete`.
+This document walks you through how to use RawKV's basic operations such as `Get`, `Put`, and `Delete`.
 
 ## Java
 
-### Step 1: import 
+### Import
 
-First of all, import all the packages used in the example.
+First of all, you need to import all necessary packages in the example.
```
Comment (TomShawn) [inline content/docs/5.1/develop/rawkv/get-put-delete.md:16]:
```suggestion
First, import all necessary packages as shown in the example.
```

File: content/docs/5.1/develop/rawkv/get-put-delete.md [hunk 5]
```diff
@@ -23,27 +23,27 @@ import org.tikv.common.TiSession;
 import org.tikv.raw.RawKVClient;
 ```
 
-`com.google.protobuf.ByteString` is used as the type of Key and Value. 
+Here, `com.google.protobuf.ByteString` is used as the type of Key and Value.
```
Comment (TomShawn) [inline content/docs/5.1/develop/rawkv/get-put-delete.md:26]:
```suggestion
In the example above, `com.google.protobuf.ByteString` is used as the type of Key and Value.
```

File: content/docs/5.1/develop/rawkv/get-put-delete.md [hunk 6]
```diff
@@ -23,27 +23,27 @@ import org.tikv.common.TiSession;
...
+To avoid conflict, `com.google.protobuf.ByteString` is shaded to `org.tikv.shade.com.google.protobuf.ByteString`, and is included in the client package.
 
-### Step 2: create RawKVClient
+### Create RawKVClient
 
-To connect to TiKV, a PD address `127.0.0.1:2379` is passed to `TiConfiguration`. 
+To connect to TiKV, a PD address `127.0.0.1:2379` is passed to `TiConfiguration`.
 
 {{< info >}}
 A comma is used to separate multiple PD addresses, e.g. `127.0.0.1:2379,127.0.0.2:2379,127.0.0.3:2379`.
```
Comment (TomShawn) [inline content/docs/5.1/develop/rawkv/get-put-delete.md:35]: Latin abbreviations are not for technical documentation.
```suggestion
A comma is used to separate multiple PD addresses. For example, `127.0.0.1:2379,127.0.0.2:2379,127.0.0.3:2379`.
```

**Comment by marsishandsome** [comment]
@TomShawn PTAL

File: content/docs/5.1/develop/rawkv/cas.md [hunk 7]
```diff
@@ -70,9 +70,9 @@ session.close();
...
 {{< warning >}}
-Users must set `conf.setEnableAtomicForCAS(true)` to ensure linearizability of `CAS` when used together with `put`, `delete`, `batch_put`, or `batch_delete`.
+You must set `conf.setEnableAtomicForCAS(true)` to ensure linearizability of `CAS` when using with `put`, `delete`, `batch_put`, or `batch_delete`.
 
 To guarantee the atomicity of CAS, write operations like `put` or `delete` in atomic mode are more expensive.
 {{< /warning >}}
 
-The code example used in this chapter can be found [here](https://github.com/marsishandsome/tikv-client-examples/blob/main/java-example/src/main/java/example/rawkv/CAS.java).
\ No newline at end of file
+The code example used in this chapter can be found [here](https://github.com/marsishandsome/tikv-client-examples/blob/main/java-example/src/main/java/example/rawkv/CAS.java).
```
Comment (TomShawn) [inline content/docs/5.1/develop/rawkv/cas.md:78]: It is not recommended to link to a personal repo.

File: content/docs/5.1/develop/rawkv/get-put-delete.md [hunk 8]
```diff
@@ -1,19 +1,19 @@
...
         weight: 1
 ---
 
-This document walks you through how to use RawKV's basic operations such as `Get`, `Put` and `Delete`.
+This document walks you through how to use RawKV's basic operations such as `Get`, `Put`, and `Delete`.
 
 ## Java
 
-### Step 1: import 
+### Import
```
Comment (TomShawn) [inline content/docs/5.1/develop/rawkv/get-put-delete.md:14]:
```suggestion
### Import packages
```

File: content/docs/5.1/develop/rawkv/get-put-delete.md [hunk 9]
```diff
@@ -23,27 +23,27 @@ import org.tikv.common.TiSession;
...
+Using the connected `org.tikv.raw.RawKVClient`, you can perform actions such as `Get`, `Put`, and `Delete`.
 
 ```java
 TiConfiguration conf = TiConfiguration.createRawDefault("127.0.0.1:2379");
 TiSession session = TiSession.create(conf);
 RawKVClient client = session.createRawClient();
 ```
 
-### Step 3: put
+### Put
```
Comment (TomShawn) [inline content/docs/5.1/develop/rawkv/get-put-delete.md:46]:
```suggestion
### Write data to TiKV
```

File: content/docs/5.1/develop/rawkv/get-put-delete.md [hunk 10]
```diff
@@ -53,7 +53,7 @@ ByteString value = ByteString.copyFromUtf8("RawKV");
 client.put(key, value);
 ```
 
-### Step 4: get
+### Get
```
Comment (TomShawn) [inline content/docs/5.1/develop/rawkv/get-put-delete.md:56]:
```suggestion
### Read data from TiKV
```

File: content/docs/5.1/develop/rawkv/get-put-delete.md [hunk 11]
```diff
@@ -63,23 +63,23 @@ assert(result.isPresent());
 assert("RawKV".equals(result.get().toStringUtf8()));
 ```
 
-### Step 5: delete
+### Delete
```
Comment (TomShawn) [inline content/docs/5.1/develop/rawkv/get-put-delete.md:66]:
```suggestion
### Delete data from TiKV
```

File: content/docs/5.1/develop/rawkv/get-put-delete.md [hunk 12]
```diff
@@ -63,23 +63,23 @@ assert(result.isPresent());
...
+Using the `delete` API, you can delete a key-value pair from TiKV.
 
 ```java
 client.delete(key);
 result = client.get(key);
 assert(!result.isPresent());
 ```
 
-### Step 6: close
+### Close
```
Comment (TomShawn) [inline content/docs/5.1/develop/rawkv/get-put-delete.md:76]:
```suggestion
### Close working instances
```

File: content/docs/5.1/develop/rawkv/cas.md (hunk 7 above)
Comment (marsishandsome) [reply content/docs/5.1/develop/rawkv/cas.md:78]: ok, I will move this code to `https://github.com/tikv/website`.

**Review by marsishandsome — APPROVED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM except for the link

**Pull request merged**